data/mods/No_Faults/modinfo.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "type": "MOD_INFO",
+    "mod-type": "SUPPLEMENTAL",
+    "ident": "no_faults",
+    "name": "Perfect Reliability",
+    "author": "Mugling",
+    "description": "All items are perfectly reliable and never spawn with faults or develop them during use",
```

Comment:
That said we could adjust the description to limit it to vehicle parts until other faults are added